editor/editor_inspector.cpp

```diff
@@ -1536,7 +1536,7 @@ void EditorInspector::update_tree() {
 
 					Color c = sscolor;
 					c.a /= level;
-					section->setup(path_name, path_name, object, c, use_folding);
+					section->setup(acc_path, path_name, object, c, use_folding);
```

Comment:
For reference, I inverted `path_name` with `acc_name` to fix section names:
![captura de tela de 2018-08-20 17-10-31](https://user-images.githubusercontent.com/1387165/44363937-0f19c280-a49c-11e8-9135-129bda90145d.png)